ibm/service/vpc/data_source_ibm_is_zones.go

```diff
@@ -35,6 +36,34 @@ func DataSourceIBMISZones() *schema.Resource {
 				Computed: true,
 				Elem:     &schema.Schema{Type: schema.TypeString},
 			},
+			isZonesInfo: {
+				Type:        schema.TypeList,
+				Computed:    true,
+				Description: "The zones information in the region",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						isZoneName: {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						isZoneUniversalName: {
+							Type:     schema.TypeString,
+							Computed: true,
+							Optional: true,
```

Comment:
Optional should be removed. Data sources have all Computed only fields

Reply:
Done